windows/CMakeLists.txt

```diff
@@ -10,6 +10,11 @@ set(SOURCES
     hidapi_hidsdi.h
 )
 
+if(EnableAsanDump)
+    # Option /MTd needed to embedd ASAN library
+    add_compile_options(-fsanitize=address /MTd)
```

Comment:
- all of: MSVC, GCC and Clang support `-fsanitize=address` so this option may be set on a level higher
- only `MTd` has to be under `if(WIN32)`